clients/cobol-lsp-vscode-extension/src/__tests__/SettingsTest.ts

```diff
@@ -18,31 +11,40 @@ import {SettingsUtils} from "../services/settings/util/SettingsUtils";
  * Contributors:
  *   Broadcom, Inc. - initial API and implementation
  */
+import * as fs from "fs-extra";
+import * as path from "path";
+import * as vscode from "vscode";
+import {C4Z_FOLDER, GITIGNORE_FILE} from "../constants";
+import {createFileWithGivenPath, initializeSettings} from "../services/Settings";
+import {SettingsUtils} from "../services/settings/util/SettingsUtils";
 
-const fsPath = "/ws-vscode";
-const c4zFolder = ".c4z";
-const fileName = ".gitignore";
+const fsPath = "tmp-ws";
+const scheme = "file";
+let wsPath: string;
+let c4zPath: string;
 let filePath: string;
 
 beforeAll(() => {
-    vscode.workspace.workspaceFolders = [{ uri: { fsPath } } as any];
-    filePath = path.join(vscode.workspace.workspaceFolders[0].uri.fsPath, c4zFolder, fileName);
+    vscode.workspace.workspaceFolders = [{uri: {fsPath}} as any];
+    wsPath = path.join(vscode.workspace.workspaceFolders[0].uri.fsPath);
+    c4zPath = path.join(wsPath, C4Z_FOLDER);
+    filePath = path.join(c4zPath, GITIGNORE_FILE);
 });
 
 afterAll(() => {
-        if (fs.existsSync(filePath)) {
-            fs.unlinkSync(filePath);
-        }
+    if (fs.existsSync(filePath)) {
```

Comment:
You check if `filePath` exists but delete `wsPath`.

Reply:
It is recursive, so starts deleting all including wsPath